webservices/tasks/legal_docs.py

```diff
@@ -30,6 +41,28 @@ def refresh():
         refresh_aos(conn)
         refresh_murs(conn)
 
+@app.task(once={'graceful': True}, base=QueueOnce)
+def reload_all_aos():
+    """
+    Reload all AOs daily (except Sunday) if there were any new or modified AOs found for the past 24 hour period
+    But on Sundays reload all AOs whether there were changes to AOs or not
+    """
+    if (datetime.date.today().strftime("%A") == 'Sunday'):
```

Comment:
Instead of burying it in this `if` statement, would it be better to have 2 separate schedules with 2 separate tasks?